lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1181,11 +1181,9 @@ def determine_group_level_monitored_resource_and_labels(tag,
       # inferred from the tag) if failed to get a monitored resource from
       # Metadata Agent with this key.
       #
-      # Docker container:
```

Comment:
I'd keep the Docker and Kubernetes headers. Otherwise it looks like container.* represents a Kubernetes container...

Reply:
Added Docker and Kubernete as inline comments.